modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/action/NotificationTemplateFTLElementsMVCResourceCommand.java

```diff
@@ -178,6 +171,9 @@ private JSONObject _getTemplateVariableGroupJSONObject(
 						content = StringBundler.concat(
 							"${", content, ".getData()}");
 					}
+					else {
+						content = StringBundler.concat("${", content, "}");
```

Comment:
Why did we removed it?

Reply:
Now we handle it with the NotificationTemplateTemplateHandler

Reply:
ok, cool!